test/xpu_api/utilities/pairs/pairs.pair/noexcept_move_assign_g.pass.cpp

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/utility>
+
+#include "support/utils.h"
+
+// #include "testsuite_struct.h" KSATODO required to remove?
```

Comment:
@rarutyun @ValentinaKats May we use this classes?
`NoexceptMoveAssignClass`, `NoexceptMoveAssignClass`
It's from test suite - see my comments in code below.

Reply:
It's funny, but all tests there has compiled with this stub code:
```C++
struct TestStruct
{
};
```